docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -83,33 +83,34 @@ EOF
 
 * You have defined the shell variables to run the script that runs the upgrade:
 +
+[subs="+quotes"]
 ----
 $ CEPH_FSID=$(oc get secret ceph-conf-files -o json | jq -r '.data."ceph.conf"' | base64 -d | grep fsid | sed -e 's/fsid = //')
 
 $ alias openstack="oc exec -t openstackclient -- openstack"
 
-$ DEFAULT_CELL_NAME="cell3" <1>
+$ *DEFAULT_CELL_NAME="cell3"*
 $ RENAMED_CELLS="cell1 cell2 $DEFAULT_CELL_NAME"
 
 $ declare -A COMPUTES_CELL1
-$ export COMPUTES_CELL1=( <2>
-> ["standalone.localdomain"]="192.168.122.100" <3>
-> # <compute1> <4>
-> # <compute2>
-> # <compute3>
+$ *export COMPUTES_CELL1=(*
+> *["standalone.localdomain"]="192.168.122.100"*
+> # *<compute1>*
+> # *<compute2>*
+> # *<compute3>*
 >)
 $ declare -A COMPUTES_CELL2
 $ export COMPUTES_CELL2=(
 > # ...
 >)
 $ declare -A COMPUTES_CELL3
-$ export COMPUTES_CELL3=(
-> # ... <5>
+$*export COMPUTES_CELL3=(*
+> # <compute1>
 >)
-># ...
+># <compute2>
 
 $ declare -A COMPUTES_API_CELL1
-$ export COMPUTES_API_CELL1=( <6>
+$*export COMPUTES_API_CELL1=(*
 > ["standalone.localdomain"]="172.17.0.100"
 > # ...
```

Comment:
Since you are replacing these, maybe add another example entry, like ``["standalone2.localdomain"]="172.17.0.101"``